### PR TITLE
Add support for docker service update

### PR DESCRIPTION
**Pull request by barrachri**
## What do these changes do?

Add support for `docker service update`.

Currently only updating the image.

## Are there changes in behavior for the user?
No

## Checklist

- [x] I think the code is well written
- [x] Unit tests for the changes exist
- [x] Documentation reflects the changes
- [x] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep alphabetical order, the file is sorted by names. 
- [x] Add a new news fragment into the `changes` folder
  * name it `<issue_id>.<type>` for example (588.bug)
  * if you don't have an `issue_id` change it to the pr id after creating the pr
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."

### CODE REVIEW

File: aiodocker/services.py [hunk 1]
```diff
@@ -87,11 +87,53 @@ def __init__(self, docker):
...
+        }
+        if rollback is True:
+            params["rollback"] = 'previous'
+
+        data = json.dumps(clean_map(spec))
+
+        await self.docker._query_json(
+            "services/{service_id}/update".format(service_id=service_id),
+            method="POST",
+            headers={"Content-type": "application/json"},
```
Comment (jettify) [inline aiodocker/services.py]: Looks like `_query_json` already adds proper content type

File: aiodocker/services.py [hunk 2]
```diff
@@ -87,11 +87,53 @@ def __init__(self, docker):
...
+                     *,
+                     image: str,
+                     rollback: bool=False
+                     ) -> bool:
+        """
+        Update a service.
+
+        Args:
+            service_id: ID or name of the service.
+            rollback: Rollback the service to teh previous service spec.
```
Comment (jettify) [inline aiodocker/services.py]: typo `teh` vs `the`

File: aiodocker/services.py [hunk 3]
```diff
@@ -87,11 +87,53 @@ def __init__(self, docker):
...
+            headers={"Content-type": "application/json"},
             data=data,
         )
         return response
 
+    async def update(self,
+                     service_id: str,
+                     *,
+                     image: str,
+                     rollback: bool=False
```
Comment (jettify) [inline aiodocker/services.py]: Probably good to have rollback test case also

**Comment by barrachri** [comment]
Thanks @jettify.

I am going to add a new test for rollback.

File: aiodocker/services.py [hunk 4]
```diff
@@ -87,11 +87,51 @@ def __init__(self, docker):
...
+        If rollback is True image will be ignored.
+
+        Args:
+            service_id: ID or name of the service.
+            rollback: Rollback the service to the previous service spec.
+
+        Returns:
+            True if successful.
+        """
+        if image is None and rollback is False:
```
Comment (asvetlov) [inline aiodocker/services.py]: Do you have any idea why the method is not covered by test?
At least codecov reports it.

**Comment by barrachri** [comment]
`rollback` doesn't work as expected....

File: aiodocker/services.py (hunk 3 above)
Comment (barrachri) [reply aiodocker/services.py]: Added.

File: aiodocker/services.py (hunk 4 above)
Comment (barrachri) [reply aiodocker/services.py]: Resolved. 👍

Comment (asvetlov) [reply aiodocker/services.py]: Cool!

**Comment by barrachri** [comment]
Thanks!

**Pull request merged**